### PR TITLE
Ak create orders

### PR DESCRIPTION
**Pull request by akintner**

### CODE REVIEW

**Comment by akintner** [comment]
closes #75

**Comment by akintner** [comment]
I think this is ready to go. Check it out @kheppenstall @tmikeschu  @wlffann

File: spec/models/order_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  context 'validations' do
+    it { is_expected.to validate_presence_of :total_cost }
+  end
+
+  context 'associations' do
+    it { should belong_to :user }
+    # it { should have_many :items, through: :order_items }
```
Comment (tmikeschu) [inline spec/models/order_spec.rb]: `has_many :items` is defined in the model, but we're planning on a join?

**Comment by tmikeschu** [comment]
Looks good! I saw you commented out some future association tests. Currently there is still an :items method in the order model. Besides that, I think it's ready.

File: spec/controllers/orders_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (kheppenstall) [inline spec/controllers/orders_controller_spec.rb]: @akintner Should we delete this file if we're not doing controller tests?

File: app/models/order.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Order < ApplicationRecord
```
Comment (kheppenstall) [inline app/models/order.rb]: We resolved by removing the `total_cost` attribute for the time being.

**Pull request merged**